test/unit/units/prepare_test.py

```diff
@@ -43,10 +46,37 @@ def test_main_raises_on_zypp_bind(
         with raises(DistMigrationZypperMetaDataException):
             main()
 
+    @patch('suse_migration_services.units.prepare.get_regionsrv_client_file_location')
     @patch('shutil.copy')
     @patch('os.listdir')
+    def test_main_raises_on_get_regionsrv_client_file_location(
+        self, mock_os_listdir, mock_shutil_copy,
+        mock_get_regionsrv_client_file_location, mock_Command_run,
+        mock_os_path_exists, mock_Fstab, mock_logger_setup,
+        mock_update_regionsrv_setup
+    ):
+        mock_os_listdir.return_value = None
+        mock_get_regionsrv_client_file_location.side_effect = \
+            [DistMigrationZypperMetaDataException('foo')]
+        mock_os_path_exists.return_value = True
+        mock_Command_run.side_effect = [
+            MagicMock(),
+            MagicMock(),
+            MagicMock(),
+            MagicMock(),
+            Exception
+        ]
+        with patch('builtins.open', mock_open(read_data='foo.susecloud.net')):
```

Comment:
as mentioned above, there should be a test with a different value that does not include `susecloud.net`